src/components/component.ts

```diff
@@ -643,7 +675,7 @@ export class Component {
   }
 
   /**
-   * Gets a Selection containing a <g> that holds the visual elements of the Component.
+   * Gets the SVG that holds the visual elements of the Component.
```

Comment:
No. There's a few futures:

1. Component lazily adds `content()` into the DOM only when it's referenced
2. Component no longer has a notion of "content()"; subclasses using SVG add SVG things themselves (this plus removal of foreground/background/boxContainer should totally rid Component of all SVG related things)
3. SVG manipulation itself is defined in its own set of classes and concepts, and SVG subclasses talk to that.

We should refactor this into (3) sooner or later